Index.html

```diff
@@ -0,0 +1,106 @@
+                    <p class="abilities">
+                          I can help you build a product , 
+                          feature or website Look through some of my work and experience! 
+                          If you like what you see and have a project you need coded,  
+                          don’t hestiate to contact me.
+                    </p>
+
+                </section>
+                <section class="inner-section">
+
+                    <hr class="vector4"/>
+                    
+                    <div class="proficiency">
+                      <div class="pf-items languages">
+                        <p class="frwk-heada">Languages</p>
+                        <ul class="ppf-head">
+                          <li>Javascript</li>
+                          <li>Ruby</li>
+                          <li>Html</li>
+                          <li>Css</li>
+                        </ul>
+                        
+                      </div>
+                      <div class="pf-items frameworks">
+                        <p class="frwk-heada">Frameworks</p>
+                        <ul class="ppf-head">
+                          <li>Bootstrap</li>
+                          <li>Ruby on Rails</li>
+                          <li>RSpec</li>
+                          <li>Capybara</li>
+                          <li>Selenium</li>
+                        </ul>
+                      </div>
+                      <div class="pf-items skills">
+                        <p class="frwk-heada">Skills</p>
+                        <ul class="ppf-head">
+                          <li>Codekit</li>
+                          <li>github</li>
+                          <li>Codepen</li>
+                          <li>Gitlab</li>
+                          <li>Terminal</li>
+                        </ul>
+                      </div>
+                    </div>
+                    
+                    <hr class="vector4"/>
+                </section>
+                <section class="inner-section heed-menu">
+                    <nav class="social">
+                        <ul class="ul-social">
+                          <li><a href="#"><img src="image/github.png" alt="github"></a></li>
+                          <li><a href="#"><img src="image/in.png" alt="linkIn"></a></li>
+                          <li><a href="#"><img src="image/twofingers.png" alt="TwuFingers"></a></li>
+                          <li><a href="#"><img src="image/Twitter.png" alt="Twitter"></a></li>
+                          <li class="m"><a href="#"><img src="image/M.png" alt="M"></a></li>
+                        </ul>
+                        <ul class="ul-work">
+                            <li><a href="#my-work"><img src="image/arrowdown.png" alt="github"></a></li>
+                            <li><a href="#my-work">Work</a></li>
+                        </ul>
+                    </nav>
+                </section>
+            </div>
+        </article>
```

Comment:
- To improve the HTML structure of your page, I suggest you use a `section` tag to wrap this section of the page instead of an `article` tag. Please fix this 👏 👏